### ISSUE TITLE
Different dim of each module

### ISSUE THREAD

**Issue by Shelsin** [GAP-LAB-CUHK-SZ/LoFA#3]
Awesome job. Thanks for authors' effort. I have a simple question. As for DiT, the dim of each module is the same. But in the paper, the author adapt SD XL as the text-to-image model. Since different dim of modules in the Unet, how to stack the model parameters?like Wan: input_params = torch.stack(input_params, dim=0).to(device, dtype=dtype)

**Comment by Jaeger416** [comment]
Hi, thanks for your interest. For handling different latent dimensions in a UNet-based architecture, you can either follow HyperDreamBooth and use the first few tokens for prediction, or design separate decoders and positional embeddings for different latent dimensions. We found that both solutions give similar performance.
